addons/website/__manifest__.py

```diff
@@ -67,6 +67,7 @@
         'views/snippets/s_carousel_intro.xml',
```

Comment:
Commit message:
- change the title by "add `s_list_group` snippet" so we can easily find the commit :slightly_smiling_face: 
- Reformulate this: `Interact with snippet just like the normal list.` => `Interact with the snippet the same way as with a normal list.`